DevOps.Util/DevOpsHttpClient.cs

```diff
@@ -49,25 +50,75 @@ internal DevOpsHttpClient(AuthorizationToken authorizationToken = default, HttpC
             HttpClient = httpClient ?? new HttpClient();
         }
 
-        internal async Task DownloadFileAsync(string uri, Stream destinationStream)
+        private async Task DownloadWithProgress(HttpResponseMessage response, Stream destinationStream)
+        {
+            Console.Write($"Downloading...");
+            using Stream contentStream = await response.Content.ReadAsStreamAsync();
+
+            long totalRead = 0L;
+            long totalReads = 0L;
+            byte[] buffer = new byte[8192];
+            const double mbdividend = 1000000.0;
+            double sizeInMbs = long.Parse(response.Content.Headers.GetValues("Content-Length").First()) / mbdividend;
```

Comment:
will it ever not return a content-length header? in that case this will throw here.

Reply:
I wouldn't expect that to happen as all files are in azure blob storage and I would expect azure blob storage responses to be complete and with that info, but I guess it wouldn't hurt to guard against that?

Reply:
I have no opinion :)